frontend/src/components/login/Login.css

```diff
@@ -0,0 +1,56 @@
+.back-image {
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+}
```

Comment:
To chyba nie jest konieczne, bo mamy tylko backgroud color.

Reply:
usunięte